ner_v2/tests/numeral/number_range/number_range_ner_tests.yaml

```diff
@@ -0,0 +1,661 @@
+tests:
+  en:
+  - _id_: en_1
+    message: I want more than 200 banana
+    outputs:
+    - max_value: null
+      min_value: '200'
+      original_text: more than 200
+      output_id: '1'
+      unit: null
+    unit_type: null
+  - _id_: en_2
+    message: My monthly salary will be more than 2k per month
+    outputs:
+    - max_value: null
+      min_value: '2000'
+      original_text: more than 2k
+      output_id: '1'
+      unit: null
+    unit_type: null
+  - _id_: en_3
+    message: more than 2.5k people in the stadium
+    outputs:
+    - max_value: null
+      min_value: '2500'
+      original_text: more than 2.5k
+      output_id: '1'
+      unit: null
+    unit_type: null
+  - _id_: en_4
+    message: 'more than 200 '
+    outputs:
+    - max_value: null
+      min_value: null
+      original_text: null
+      output_id: '1'
+      unit: null
+    unit_type: currency
+  - _id_: en_5
+    message: more than 2k
+    outputs:
+    - max_value: null
+      min_value: null
+      original_text: null
+      output_id: '1'
+      unit: null
+    unit_type: currency
+  - _id_: en_6
+    message: more than 2.5k
+    outputs:
+    - max_value: null
+      min_value: null
+      original_text: null
+      output_id: '1'
+      unit: null
+    unit_type: currency
+  - _id_: en_7
+    message: more than 200 rupees
+    outputs:
+    - max_value: null
+      min_value: '200'
+      original_text: more than 200 rupees
+      output_id: '1'
+      unit: rupees
+    unit_type: currency
+  - _id_: en_8
+    message: more than 2k rupees
+    outputs:
+    - max_value: null
+      min_value: '2000'
+      original_text: more than 2k rupees
+      output_id: '1'
+      unit: rupees
+    unit_type: currency
+  - _id_: en_9
+    message: more than 2.5k rupees
+    outputs:
+    - max_value: null
+      min_value: '2500'
+      original_text: more than 2.5k rupees
+      output_id: '1'
+      unit: rupees
+    unit_type: currency
+  - _id_: en_10
+    message: 200 to 300
+    outputs:
+    - max_value: '300'
+      min_value: '200'
+      original_text: 200 to 300
+      output_id: '1'
+      unit: null
+    unit_type: null
+  - _id_: en_11
+    message: 200 – 300
+    outputs:
+    - max_value: '300'
+      min_value: '200'
```

Comment:
min_value and max_value are str, is this correct ?

Reply:
Yes.